scvi/metrics/log_likelihood.py

```diff
@@ -4,13 +4,18 @@
 import torch.nn.functional as F
 
 
-def compute_log_likelihood(vae, data_loader):
+def compute_log_likelihood(vae, data_loader, name):
```

Comment:
Better not to take `name` as an argument. Instead, let's make `compute_log_likelihood`  a method in the `VariationalInference` class. Then override `compute_log_likelihood` in the `VariationalInferenceFish` class.